kong/tools/utils.lua

```diff
@@ -37,6 +37,8 @@ local re_match      = ngx.re.match
 local inflate_gzip  = zlib.inflateGzip
 local deflate_gzip  = zlib.deflateGzip
 local stringio_open = pl_stringio.open
+local now           = ngx.now
+local update_time   = ngx.update_time
```

Comment:
These two lines are only used by `get_updated_now_ms`, 
we should move these two lines into a `do end` block,
like `sha256_hex` did.